src/azul/service/hca_response_v5.py

```diff
@@ -254,26 +241,12 @@ class KeywordSearchResponse(AbstractResponse, EntryFetcher):
     Not to be confused with the 'keywords' endpoint
     """
 
-    def _merge(self, dict_1: MutableJSON, dict_2: MutableJSON, identifier):
-        merged_dict: MutableMapping[str, List[AnyMutableJSON]] = defaultdict(list)
-        dict_id = dict_1.pop(identifier)
-        dict_2.pop(identifier)
-        for key, value in chain(dict_1.items(), dict_2.items()):
-            if isinstance(value, str):
-                merged_dict[key] = list(set(merged_dict[key] + [value]))
-            elif isinstance(value, int):
-                merged_dict[key] = list(merged_dict[key] + [value])
-            elif isinstance(value, list):
-                cleaned_list = list(filter(None, chain(value, merged_dict[key])))
-                if len(cleaned_list) > 0 and isinstance(cleaned_list[0], dict):
-                    # Make each dict hashable so we can deduplicate the list
-                    merged_dict[key] = list(map(thaw, set(map(freeze, cleaned_list))))
-                else:
-                    merged_dict[key] = list(set(cleaned_list))
-            elif value is None:
-                merged_dict[key] = []
-        merged_dict[identifier] = dict_id
-        return dict(merged_dict)
+    @classmethod
+    def sorted(cls, value):
```

Comment:
What kinds of types do you expect? AFAICT, only None or list or set are possible. This method should fail if anything else is passed to it as that constitutes an error. Sounds familiar?

Reply:
Many various value types will pass through this function including `int` (donorCount, totalCells) and `str` (projectTitle, modelOrgan). While the `int` values might always be `int`, the `str` values such as projectTitle and projectShortname will appear as either `str` or a list of `str`

Reply:
Can you demonstrate this to me?

Reply:
Yes, a request to `/repository/projects` results in `make_projects` being sent hits that contain `hit['contents']['projects']['project_title']` values that are `str`,